test/unit/processing/index.test.js

```diff
@@ -24,11 +25,17 @@ describe('start processing', () => {
     expect(schedulePendingSettlements).toHaveBeenCalled()
   })
 
-  test('should call schedulePendingSettlements once', async () => {
+  test('should call schedulePendingSettlements once if construction active', async () => {
     await processing.start()
     expect(schedulePendingSettlements).toHaveBeenCalledTimes(1)
   })
 
+  test('should not call schedulePendingSettlements if construction not active', async () => {
+    processingConfig.constructionActive = false
+    await processing.start()
+    expect(schedulePendingSettlements).not.toHaveBeenCalled()
+  })
+
```

Comment:
If the intention is to always call `setTimeout` after each  `processingConfig.settlementProcessingInterval`, then, I think you need to add a test to confirm that `setTimeout` is called when `processingConfig.constructionActive = false`